PowerShellGet/PSGet.Resource.psd1

```diff
@@ -21,6 +21,7 @@ ConvertFrom-StringData @'
         VersionRangeAndRequiredVersionCannotBeSpecifiedTogether=You cannot use the parameters RequiredVersion and either MinimumVersion or MaximumVersion in the same command. Specify only one of these parameters in your command.
         RequiredVersionAllowedOnlyWithSingleModuleName=The RequiredVersion parameter is allowed only when a single module name is specified as the value of the Name parameter, without any wildcard characters.
         MinimumVersionIsGreaterThanMaximumVersion=The specified MinimumVersion '{0}' is greater than the specified MaximumVersion '{1}'.
+        AllowPrereleaseRequiredToUsePrereleaseStringInVersion=The AllowPrerelease parameter must be specified when using the Prerelease string in MinimumVersion, MaximumVersion, or RequiredVersion. 

```

Comment:
Consider putting parameter name in commas. like '-AllowPrerelease'

Reply:
I think, Fareha meant single quotes here.

---
In reply to: [138749377](https://github.com/rebro-msft/PowerShellGet/pull/2#discussion_r138749377) [](ancestors = 138749377)